cfme/common/provider.py

```diff
@@ -310,7 +310,7 @@ def validate_stats(self, ui=False):
                      {'ui': ui},
                      message="do_stats_match_db",
                      num_sec=1000,
-                     delay=60)
+                     delay=120)
```

Comment:
Ahh ok, so, we need to update wrapanapi to include a parameter to get the images from all places, then  modify the test to versionpick and either include that parameter in the list_images call, or not.